### PR TITLE
add form template documentation (take 2)

### PR DESCRIPTION
**Pull request by carodew**
Second attempt to add documentation to: 
* sign in form (also updates all references to "log in/login" to "sign in/sign-in"
* password reset form (also adds guidance to customize password requirements)
* address form

### CODE REVIEW

File: _components/form-templates.md [hunk 1]
```diff
@@ -158,14 +158,7 @@ lead: Patterns for some of the most commonly used forms on government websites.
...
   <div id="collapsible-0" aria-hidden="false" class="usa-accordion-content">
-    <h4 class="usa-heading">Use</h4>
-    <p>As you customize this form, ensure it continues to:</p>
-    <ul class="usa-content-list">
-      <li>Label the optional ones. Users can infer that all the others are required.</li>
-      <li>Support both 5 and 9 digit zips. Some addresses require a nine digit ZIP code. The input mask should be "#####-####"" so that the text is properly formatted, regardless of whether a user enters a 5 or 9 digit zip code. However, this input mask should only appear when people click away from the ZIP code input box (see the accessibility guidelines).</li>
-    </ul>
-
-    <h4 class="usa-heading">Accessibility</h4>
+   <h4 class="usa-heading">Accessibility</h4>
```
Comment (maya) [inline _components/form-templates.md]: the indentation is off, please align with the others.

File: _components/form-templates.md [hunk 2]
```diff
@@ -175,15 +168,31 @@ lead: Patterns for some of the most commonly used forms on government websites.
     <ul class="usa-content-list">
       <li>Make sure any input masking is accessible to people using screen readers. We use <a href="https://github.com/filamentgroup/politespace">Filament Group's Politespace</a> to ensure a good experience.</li>
     </ul>
+    <h4 class="usa-heading">Usability</h4>
+    <h5>When to use</h5>
+    <ul class="usa-content-list">
+      <li>When you need to be able to parse out the specific parts of a mailing address. </li>
```
Comment (maya) [inline _components/form-templates.md]: please remove the extra white space before the closing `</li>`

File: _components/form-templates.md [hunk 3]
```diff
@@ -175,15 +168,31 @@ lead: Patterns for some of the most commonly used forms on government websites.
...
+    <ul class="usa-content-list">
+      <li>Only label the optional inputs. Users can infer that all the others are required.</li>
+      <li>If possible, let users type their state’s abbreviation when they reach the “state” drop down menu.</li>
+      <li>Support both 5 and 9 digit ZIP codes.  Some addresses require a nine digit ZIP code. The input mask should be “#####-####” so that the text is properly formatted, regardless of whether a user enters a 5 or 9 digit ZIP code. </li>
+    </ul>
   </div>
 </div>
 
-<h2 class="usa-heading" id="login-form">Login form</h2>
+<h2 class="usa-heading" id="login-form">Sign-in form</h2>
```
Comment (maya) [inline _components/form-templates.md]: This is not consistent with what's below - Sign-in vs. Sign in

File: _components/form-templates.md [hunk 4]
```diff
@@ -230,7 +230,24 @@ lead: Patterns for some of the most commonly used forms on government websites.
     </ul>
     <p>We also recommend:</p>
     <ul class="usa-content-list">
-      <li>Don’t automatically log a user out without giving them 20 seconds advance notice to request more time. Users with disabilities sometimes require more time to respond to prompts.</li>
+      <li>Don’t automatically sign out a user without giving them 20 seconds advance notice to request more time. Users with disabilities sometimes require more time to respond to prompts.</li>
+    </ul>
+        <h4 class="usa-heading">Usability</h4>
```
Comment (maya) [inline _components/form-templates.md]: indentation is off, please fix

File: _components/form-templates.md [hunk 5]
```diff
@@ -286,24 +303,23 @@ lead: Patterns for some of the most commonly used forms on government websites.
...
-      <li>People have an easier time remembering their email address rather than a unique username, so allow them to use their email address to log in. However, some people don’t have an email address, so don’t let this be the only option.</li>
-      <li>When a Sign in form is required, consider allowing users to stay logged in ("Remember me") on trusted computers so they can avoid this barrier in the future.</li>
-      <li>Make it easy to retrieve a forgotten username and password. Most authentication failures occur because a user has forgotten their username or password. This is especially common when a long time passes between visits, as is the case with most federal websites.</li>
-      <li>Password masking (replacing what the user types with a generic symbol) makes it more likely that users will make mistakes when trying to log in without offering much in additional security. Allow users to unmask this field so they can see what they type. This is especially useful on mobile, when mistakes are more likely.</li>
-    </ul>
-
-    <h4 class="usa-heading">Accessibility</h4>
+   <h4 class="usa-heading">Accessibility</h4>
```
Comment (maya) [inline _components/form-templates.md]: indentation is off, please fix.

File: _components/form-templates.md (hunk 1 above)
Comment (carodew) [inline _components/form-templates.md]: I don't know how to do that within GitHub.

File: _components/form-templates.md (hunk 3 above)
Comment (carodew) [inline _components/form-templates.md]: Sign-in as an adjective or a noun. Use the sign-in form to access your account. 
Sign in as a verb. Please sign in to access your account.

File: _components/form-templates.md (hunk 1 above)
Comment (maya) [inline _components/form-templates.md]: It should be two spaces for each "tab". Make sure you are using spaces and it should be fixed.

**Comment by carodew** [comment]
Should be fixed in 9dd3c0b.

**Comment by maya** [comment]
@carodew line 55 says "Use" should this be Usability instead?

**Comment by carodew** [comment]
Good catch. fixed in  7a9b6e9

**Pull request merged**